maker-frontend/src/MakerApp.tsx

```diff
@@ -165,6 +165,8 @@ export default function App() {
                                         max_quantity: Number.parseFloat(maxQuantity),
                                         // TODO: Populate funding rate from the UI
                                         funding_rate: (0.00002283 * 24), // annualized 20% by default to have some values
+                                        // TODO: Populate funding rate from the UI
+                                        daily_funding_rate_long: (-0.00002283 * 24), // annualized 20% by default to have some values
```

Comment:
On purpose negative?

Reply:
yes, wanted to visualise the maker profiting both ways - happy to remove the negative sign if it's more misleading than helping!

Reply:
no worries, this is just for testing anyways :)